python/overrides.js

```diff
@@ -15,6 +15,9 @@
   
     'should waitBeforeCapture in open': { skipEmit: true },
     'should waitBeforeCapture in check': { skipEmit: true },
+    'should waitBeforeCapture with breakpoints in check': { skipEmit: true },
+    'should waitBeforeCapture with breakpoints in open': { skipEmit: true },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9117994)
```suggestion
    "should waitBeforeCapture with breakpoints in open": { skipEmit: true },
```